application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/views/stacktrace/StacktraceView.java

```diff
@@ -908,8 +913,11 @@ public String getText(Object element) {
 			IQuantity duration = getDurationCount(frame.getItems());
 			IQuantity totalDuration = getDurationCount(
 					getRootFork(frame.getBranch().getParentFork()).getAllItemsInFork());
-			return UnitLookup.PERCENT_UNITY.quantity(duration.doubleValue() / (double) totalDuration.doubleValue())
-					.displayUsing(IDisplayable.AUTO);
+			if (duration.doubleValue() > 0) {
+				return UnitLookup.PERCENT_UNITY.quantity(duration.doubleValue() / (double) totalDuration.doubleValue())
+						.displayUsing(IDisplayable.AUTO);
+			} else
```

Comment:
Style nit: Please also use braces for the else part.

Reply:
Done.